crates/tendermint/src/chain/info.rs

```diff
@@ -0,0 +1,32 @@
+use {block, chain, timestamp::Timestamp};
+
+/// Information about a particular Tendermint blockchain
+#[derive(Clone, Debug)]
+pub struct Info {
```

Comment:
I didn't have a particular use for this, so it's something of a strawman, but I thought it would be useful for the KMS to start tracking its view of this information and created the type I wanted to use for that. It seems like there's a lot of additional information we could potentially track as well.